test/fixtures/todo/json5/serialization/object.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = [
+  {
+    name: 'object with numeric literal as key',
+    value: { 42: true },
+    serialized: '{42:true}'
+  },
+  {
+    name: 'object with unquoted keys',
```

Comment:
This name was taken from corresponding test for JSON5 parsing test: `test/fixtures/json5/objects/unquoted-keys.json5`.